src/objects/InstancedMesh.js

```diff
@@ -39,15 +39,9 @@ InstancedMesh.prototype = Object.assign( Object.create( Mesh.prototype ), {
 
 	},
 
-	setColorAt: function ( index, color ) {
-
-		if ( this.instanceColor === null ) {
-
-			this.instanceColor = new BufferAttribute( new Float32Array( this.count * 3 ), 3 );
+	getColorAt: function ( index, color ) {
 
-		}
-
-		color.toArray( this.instanceColor.array, index * 3 );
+		color.fromArray( this.instanceColor.array, index * 3 );
```

Comment:
@mrdoob Should we make this more safe and check each time whether `instanceColor` is `null` or not?

Reply:
Hmm, I think we can leave it without the check for now and we can add it if we see people struggling with it.